<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainView</class>
 <widget class="QWidget" name="MainView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>527</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="tabSpacing">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>754</width>
        <height>446</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="icon">
       <iconset resource="Resources.qrc">
        <normaloff>:/icons/input.ico</normaloff>:/icons/input.ico</iconset>
      </attribute>
      <attribute name="label">
       <string>Receipe In</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="page2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>754</width>
        <height>446</height>
       </rect>
      </property>
      <attribute name="icon">
       <iconset resource="Resources.qrc">
        <normaloff>:/icons/output.ico</normaloff>:/icons/output.ico</iconset>
      </attribute>
      <attribute name="label">
       <string>Receipe Out</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="page3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>754</width>
        <height>446</height>
       </rect>
      </property>
      <attribute name="icon">
       <iconset resource="Resources.qrc">
        <normaloff>:/icons/xml-file.ico</normaloff>:/icons/xml-file.ico</iconset>
      </attribute>
      <attribute name="label">
       <string>POHODA - Export/Import log</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTextEdit" name="textEdit">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
